thrust/thrust/tuple.h

```diff
@@ -39,13 +39,7 @@
 #  pragma system_header
 #endif // no system header
 
-#include <thrust/type_traits/is_trivially_relocatable.h>
-
```

Comment:
I think we need to keep this, so `::cuda::std::tuple` and `::cuda::std::pair` stay proclaimed trivially relocatable.

Reply:
Nope, that is already handled in `thrust/type_traits/is_trivially_relocatable.h` itself

Reply:
But if my TU is including `thrust/pair.h`, then `type_traits/is_trivially_relocatable.h` must be included as well, otherwise I don't get the specialization.

Reply:
Oh wait, but then I cannot query the trait either. All good, keep it!